packages/exec/__tests__/exec.test.ts

```diff
@@ -572,6 +732,42 @@ describe('@actions/exec', () => {
       )
     })
 
+    it('execs .cmd from path (Windows)', async () => {
```

Comment:
here is the test related to the bug fix.

toolrunner should which the tool before attempting to invoke. otherwise simple commands like `npm --version` fail on windows (ENOENT) because `npm` resolves to `npm.cmd` and node doesnt find it.